app/components/profile-nav.js

```diff
@@ -61,4 +63,17 @@ export default Component.extend({
     return !this.features.get('enterpriseVersion') && !!billingEndpoint;
   }),
 
+  didRender() {
+    const allowance = this.model.allowance;
+
+    if (!allowance || allowance.subscription_type !== 2)
+      return;
+
+    if (!allowance.private_repos) {
+      this.flashes.custom('flashes/negative-balance-private', { owner: this.model, isUser: this.model.isUser });
+    } else if (!allowance.public_repos) {
+      this.flashes.custom('flashes/negative-balance-public', { owner: this.model, isUser: this.model.isUser });
```

Comment:
It feels like this action also needs an opposite one. When user has a negative balance, this flash appears. User sees the message, goes to the Plan page, replenishes the balance, but still sees the message, because it never gets removed before page reload. 

Also, how would it work when user visits the profile section multiple times? Will it add multiple flashes? I mean, I know, that it will _render_ only the first one, but will they stack up there? Probably it would make sense to implement some kind of a simple management for these notifications (f.e. clear them when user leaves the page)?